References.md

```diff
@@ -8,6 +8,7 @@
 -nodejs documentation
 -MDN js documentation
 -npm documentation
+-https://lord.io/blog/2014/travis-multiple-subdirs/
```

Comment:
@sperry94 
Could we move References.md to the Wiki at some point? We're starting to have a lot of .mds in the repo, and it could be cleaner to keep them all in the Wiki. Doesn't have to be with this PR, but maybe in a future one :)